sklearn/utils/validation.py

```diff
@@ -232,25 +234,27 @@ def _ensure_sparse_format(spmatrix, accept_sparse, dtype, copy,
     spmatrix_converted : scipy sparse matrix.
         Matrix that is ensured to have an allowed type.
     """
-    if accept_sparse is None:
+    if accept_sparse in [None, False]:
```

Comment:
Alternatively we could have just added support `accept_sparse='any'`. But True / False are fine too in my opinion.